CERT-Polska/mquery
# Add a button to copy all hashes

**Your checklist for this pull request**

**What is the current behaviour?**
No way to copy all the hashes.

**What is the new behaviour?**
There is a button in the matches view to copy all hashes:

![ss](https://user-images.githubusercontent.com/7026881/144125771-14d471ab-a75b-4dac-aa94-16106ec1971e.png)

**Test plan**
- Run a yara query with at least one result.
- Click the button.
- Clipboard content should change.

**Closing issues**

fixes #34
